Mathlib/Combinatorics/SimpleGraph/Matching.lean

```diff
@@ -89,6 +90,12 @@ lemma IsMatching.map_ofLE (h : M.IsMatching) (hGG' : G ≤ G') :
   use w
   simpa using hv' ▸ hw
 
+lemma IsMatching.not_adj_of_ne {M : Subgraph G} {u v w : V} (hM : M.IsMatching) (huv : v ≠ w)
+    (hadj : M.Adj u v) : ¬M.Adj u w := by
```

Comment:
Can we have the other side version of that? And maybe rename to
```suggestion
lemma IsMatching.not_adj_left_of_ne {M : Subgraph G} {u v w : V} (hM : M.IsMatching) (huv : v ≠ w)
    (hadj : M.Adj u v) : ¬M.Adj u w := by
```

Reply:
Done:)